packages/catalog-process/test/db.test.ts

```diff
@@ -1370,14 +1381,60 @@ describe("database test", async () => {
           ),
           docs: [expectedDocument],
         };
+        const currentDate = new Date();
         const expectedEService: EService = {
           ...eService,
           id: writtenPayload.eService!.id,
-          name: `${eService.name} - clone`,
+          name: `${eService.name} - clone - ${currentDate.toLocaleDateString(
+            "it-IT"
+          )} ${currentDate.toLocaleTimeString("it-IT")}`,
           descriptors: [expectedDescriptor],
           createdAt: new Date(Number(writtenPayload.eService?.createdAt)),
         };
         expect(writtenPayload.eService).toEqual(toEServiceV1(expectedEService));
+        vi.useRealTimers();
+      });
+      it("should throw eServiceDuplicate if an eService with the same name already exists", async () => {
+        vi.useFakeTimers();
+        vi.setSystemTime(new Date());
+
+        const descriptor: Descriptor = {
+          ...mockDescriptor,
+          state: descriptorState.draft,
+          interface: mockDocument,
+          docs: [mockDocument],
+        };
+        const eService1: EService = {
+          ...mockEService,
+          id: generateId(),
+          descriptors: [descriptor],
+        };
+        await addOneEService(eService1, postgresDB, eservices);
+
+        const currentDate = new Date();
+
+        const conflictEServiceName = `${
+          eService1.name
+        } - clone - ${currentDate.toLocaleDateString(
+          "it-IT"
+        )} ${currentDate.toLocaleTimeString("it-IT")}`;
```

Comment:
Same for the previous

Reply:
Makes sense. I have made an improvement to make the test more readable.